pydoctor/templates/attribute-child.html

```diff
@@ -1,4 +1,5 @@
 <div xmlns:t="http://twistedmatrix.com/ns/twisted.web.template/0.1" class="function">
+  <meta name="template" content="pydoctor-default" major_version="1" minor_version="0" />
```

Comment:
If we add new template tag in the twisted templates, they won't be filled (missing information) if the custom template is outdated. 

> Do we need a major and minor version 

Probably not.

> `<meta name="pydoctor-template-version" content="1" />`

This will work too